content/doc/developer/plugin-development/choosing-jenkins-baseline.adoc

```diff
@@ -14,7 +14,7 @@ You need to balance compatibility and features:
 In particular, the LTS Release Line is based on slightly older releases to provide a more stable experience for conservative users.
 * *Building against recent Jenkins versions* allows you to use recently added core features and API from your plugin. 
 You will also use that newer version for `mvn hpi:run`, so it may be easier to test your plugin with newer Jenkins releases. 
-* *Do not use versions no longer supported by the update center*, which is currently anything older than 2.372 for weekly releases, and 2.361.2 for LTS releases.
+* *Do not use versions no longer supported by the update center*, which is currently anything older than 2.372 for weekly releases, and 2.361.4 for LTS releases.
```

Comment:
There is a complication here, but I'm not sure we should be too concerned about that complication.  This text is inaccurate with this update because the text is describing the versions no longer supported by the update center.  Version are dropped from support by the update center at a specific time since release (15 months, I believe).  Thus, the text in this location should remain "2.361.2"